### ISSUE TITLE
Train openpose with nan loss

### ISSUE THREAD

**Issue by NokiaDDT** [CVBox/PyTorchCV#8]
Hi, @CVBox ,

I follow the following instructions to train openpose model and got Loss = nan for all iterations, is this a bug?

git commit 1cb7378c

Step 1. cd $PytorchCV/datasets/pose/coco; bash coco_pose_generator.sh
Step 2. cd $PytorchCV; python main.py  --hypes hypes/pose/coco/op_coco_pose.json --base_lr 0.001 --phase train

2018-0615 19:52:19,065 INFO    [open_pose.py, 131] Loss Heatmap:nan, Loss Asso: nan
2018-0615 19:52:19,065 INFO    [open_pose.py, 139] Train Epoch: 1   Train Iteration: 800 Time 77.872s / 100iters, (0.779) Data load 0.016s / 100iters, (0.000163)
Learning rate = [0.001, 0.002, 0.004, 0.008]    Loss = nan (ave = nan)
